autopilot/contractor.go

```diff
@@ -399,7 +399,7 @@ func (c *contractor) runContractRenewals(cfg Config, blockHeight, currentPeriod
 
 func (c *contractor) runContractFormations(cfg Config, blockHeight, currentPeriod uint64, budget *types.Currency, renterAddress types.UnlockHash) ([]types.FileContractID, error) {
 	// fetch all active contracts
-	active, err := c.ap.bus.Contracts()
+	active, err := c.ap.bus.Contracts("all")
```

Comment:
I'm a bit torn on whether or not I like this but either way this should be a constant.